src/snapred/ui/view/InitializeCalibrationCheckView.py

```diff
@@ -0,0 +1,70 @@
+from PyQt5.QtCore import *
+from PyQt5.QtGui import *
+from PyQt5.QtWidgets import *
```

Comment:
Change these from wildcard imports to specific imports, I would have thought precommit would have caught that.